docs/source/configuration/persisted-queries.mdx

```diff
@@ -0,0 +1,72 @@
+---
+title: Operation safelisting with Persisted Queries
+description: With GraphOS Enterprise
+---
+
+> ⚠️ **Persisted queries is an [Enterprise feature](https://www.apollographql.com/blog/platform/evaluating-apollo-router-understanding-free-and-open-vs-commercial-features/) of the Apollo Router.** It requires an organization with a [GraphOS Enterprise plan](https://www.apollographql.com/pricing/).
+>
+> If your organization _doesn't_ currently have an Enterprise plan, you can test out this functionality by signing up for a free [Enterprise trial](https://www.apollographql.com/docs/graphos/org/plans/#enterprise-trials).
+
+Persisted Queries gives you the tools to prevent unwanted traffic from reaching your graph.
+
+It has two modes of operation:
```

Comment:
Doesn't it have 4 modes of operating? 

- preview_persisted_queries/enabled: true
- preview_persisted_queries/enabled: true, log_unknown: true
- safelist/enabled:true
- safelist/enabled:true + require_id: true

Reply:
Handled in https://github.com/apollographql/router/pull/3501